presto-client/src/main/java/com/facebook/presto/client/ServerInfo.java

```diff
@@ -30,15 +31,19 @@
     private final String environment;
     private final boolean coordinator;
 
+    private final Duration uptime;
+
     @JsonCreator
     public ServerInfo(
             @JsonProperty("nodeVersion") NodeVersion nodeVersion,
             @JsonProperty("environment") String environment,
-            @JsonProperty("coordinator") boolean coordinator)
+            @JsonProperty("coordinator") boolean coordinator,
+            @JsonProperty("uptime") Duration uptime)
     {
         this.nodeVersion = requireNonNull(nodeVersion, "nodeVersion is null");
         this.environment = requireNonNull(environment, "environment is null");
         this.coordinator = requireNonNull(coordinator, "coordinator is null");
+        this.uptime = requireNonNull(uptime, "uptime is null");
```

Comment:
This will break if a newer client is used against an old server. This is problematic since this is used by the JDBC driver.